packages/logger/src/interfaces/Reporter.ts

```diff
@@ -10,5 +10,5 @@ export interface Reporter<TLogStructuredData extends LogStructuredData = LogStru
   log(entry: LogEntry<TLogStructuredData>): void;
 
   /** renders a summary based on the incoming data */
-  summarize<TSummaryData>(context: TSummaryData): void;
+  summarize(context: unknown): void;
```

Comment:
curious if you're running api extractor . You're changing typings as a "patch" here.

Reply:
But because there's no api file changes, I would assume this isn't exported.

Reply:
This IS exported. But this is actually a compatible change (that context TSummaryData is an unknown generic type known only at the time you implement the Reporter). So I'm just making it simpler here...

Adding Api-Extractor would be good here (though not strictly necessary *currently* because these packages aren't used by lage v2 yet)